lib/CMakeLists.txt

```diff
@@ -80,7 +80,6 @@ add_library(lib${PROJECT_NAME} $<TARGET_OBJECTS:libprojectsrc>)
 set_target_properties(lib${PROJECT_NAME} PROPERTIES VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 target_link_libraries(lib${PROJECT_NAME}
     ${LEATHERMAN_LIBRARIES}
-    ${Boost_LIBRARIES}
```

Comment:
It surprised me too. This one fixed one of the multiple definitions errors on Windows when linking Facter against cpp-hocon...